Vagrantfile

```diff
@@ -1,3 +1,9 @@
+$puppet_update_to_37 = <<SCRIPT
+
+gem install puppet --version "=3.7.0" --no-ri --no-rdoc
```

Comment:
I'm not so fond of installing puppet via gems, for the possible conflicts with packaged puppet.
Would it be better to have a shell provisioner that installs puppetlabs repos and installs a recent puppet version using the local package provider?